frontend/src/components/orders/OfferSummary.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import Card from "primevue/card";
+import Dropdown from "primevue/dropdown";
+import { computed, ref, watch } from "vue";
+
+import { updateOffer } from "@/backendClient";
+import useNotifications from "@/compositions/useNotifications";
+import { ArticleKind, OfferStatus } from "@/global/types/appTypes";
+import type { Offer } from "@/global/types/entities";
+import { calculateBrutto, calculatePrice } from "@/helpers/utils";
+
+const props = defineProps<{
+  offer: Offer;
+}>();
+const notifications = useNotifications();
+
+const offersType = Object.values(OfferStatus);
+
+let offerStatusValue = ref<OfferStatus>(props.offer.status);
+const allItemsSum = computed(() => {
+  return calculatePrice(props.offer.items, props.offer.offered_at);
+});
+
+watch(offerStatusValue, async () => {
+  await updateOffer(props.offer.id, {
+    status: offerStatusValue.value,
+  });
```

Comment:
I think this is a bit dangerous. The user might (mistakenly) update the status without the intention to save it.

Can we make the action more explicit? We could add a "Save/Update Offer" button or maybe we can add a confirmation before updating the status?